test/lib/usd/translators/testUsdExportMaterialX.py

```diff
@@ -182,7 +182,7 @@ def testExportTexturedMaterialXpPlane1(self):
                          "ND_standard_surface_surfaceshader")
         mxNode = Sdr.Registry().GetShaderNodeByIdentifier("ND_standard_surface_surfaceshader")
         if mxNode:
-            self.assertEqual(outputName, mxNode.GetOutputNames()[0])
+            self.assertEqual(outputName, mxNode.GetShaderOutputNames()[0])
```

Comment:
That API was part of Ndr shader, which was removed in 25.08. The Sdr shader API should be used instead and has been there for a long time.